lib/u3d/installer.rb

```diff
@@ -132,13 +132,16 @@ def install_pkg(file_path, version: nil, target_path: nil)
       target_path ||= DEFAULT_MAC_INSTALL
       command = "installer -pkg #{file_path.shellescape} -target #{target_path.shellescape}"
       unity = installed.find { |u| u.version == version }
+      temp_path = File.join(target_path, 'Applications', 'Unity')
       if unity.nil?
         UI.verbose "No Unity install for version #{version} was found"
         U3dCore::CommandExecutor.execute(command: command, admin: true)
+        destination_path = File.join(target_path, 'Applications', UNITY_DIR % version)
+        Utils.ensure_dir destination_path
+        FileUtils.mv temp_path, destination_path
```

Comment:
I think this will not work the `ensure_dir` call will create
```
/Applications/Unity_x.x.x.x//Unity
```

Reply:
Tested and it doesn't work as you expected it